src/classic.rs

```diff
@@ -1,25 +1,33 @@
-use image::{Rgba, RgbaImage};
+use std::io;
 use hsl::HSL;
-use {hsl_to_rgba, fill_rect, rgba};
+use pixelate::{Color, Image, Error};
 
-fn white() -> Rgba<u8> {
-	rgba(255, 255, 255, 255)
+use crate::util::{create_image_data, hsl_to_rgb};
+
+pub struct Classic {
```

Comment:
Would be good to add docs to pubic items (but since they were not here earlier it's a separate issue)

Reply:
Added doc comments to both structs.